consensus/replay_test.go

```diff
@@ -80,19 +80,20 @@ func TestReplayCatchup(t *testing.T) {
 	// start timeout and receive routines
 	cs.startRoutines(0)
 
+	// 	cs.scheduleRound0(cs.Height)
```

Comment:
should just delete that. schedule round happens after the catchup (https://github.com/tendermint/tendermint/blob/master/consensus/state.go#L308).